control-plane-operator/controllers/hostedcontrolplane/kas/params.go

```diff
@@ -116,6 +124,12 @@ func NewKubeAPIServerParams(ctx context.Context, hcp *hyperv1.HostedControlPlane
 		params.Image = hcp.Spec.Configuration.Image
 		params.Scheduler = hcp.Spec.Configuration.Scheduler
 	}
+	if reqInflight := hcp.Annotations[hyperv1.KubeAPIServerMaximumRequestsInFlight]; reqInflight != "" {
+		params.MaxRequestsInflight = reqInflight
```

Comment:
Do we need to validate the param is correct before put the annotation content directly as a value?